ar.xjuan.Cambalache.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id" : "ar.xjuan.Cambalache",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "separate-locales": false,
+    "command" : "cambalache",
+    "finish-args" : [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--device=dri"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "cambalache",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url": "https://gitlab.gnome.org/jpu/cambalache.git",
+   		    "tag": "0.7.3",
+                    "commit": "0022230d8a3c0756347046d2a7d160a2cd3faf07"
```

Comment:
> Ideally I would like to track main branch instead of using a tag and once I make a stable release add it to regular flathub

It's fine to specify only a commit here. We will merge it to branch called beta, which will publish it to flathub-beta, and when it's ready, you can push to the master branch to publish it to regular Flathub.

Reply:
Right, I just like having the tag too because the commit does not mean anything to me, with the tag I know right out of the bat which version is at

Reply:
tag + commit is perfectly fine. (to me it's preferred)
it protects against retagging.